src/popen.rs

```diff
@@ -607,7 +607,6 @@ mod os {
     use std::ffi::OsString;
     use std::time::{Duration, Instant};
 
-    use super::ChildState::*;
```

Comment:
Unfortunately, omitting this line prevents successful compilation with Rust 1.14.0 (stable). This is visible in the Travis CI log.